src/analytics/Events.tsx

```diff
@@ -26,6 +26,9 @@ export enum AppEvents {
   in_app_review_error = 'in_app_review_error',
 
   handle_deeplink = 'handle_deeplink',
+
+  gas_fee_warning_impression = 'gas_fee_warning_impression',
+  gas_fee_warning_cta_press = 'gas_fee_warning_cta_press',
```

Comment:
nit: move this to `FeeEvents`? I think the two events currently there may not even be used anymore though, so could just replace these with those.